src/versions.py

```diff
@@ -1,12 +1,44 @@
-api = "d6a8f23"
-reporting_api = "92bd655"
-db = "139756a"
-contrib_portal = "2744c82"
-admin_portal = "2744c82"
-report_portal = "2744c82"
-contrib_portal = "56df438"
-admin_portal = "56df438"
-report_portal = "56df438"
-cert_tool = "ae10ceb"
-proxy = "623c7be"
-orderly = "c307429"
+import re
+from os.path import join
+from subprocess import run, PIPE
+
+
+def get_submodule_version(path):
```

Comment:
A possibly simpler way of achiving this:

```
git -C backup rev-parse --short HEAD <path>
```

will return the short hash for the path

Reply:
Sorry this should be `git -C <path> rev-parse --short HEAD`

Reply:
Again, a much better way. Clearly Googling "how to do X in Git" is not the way forwards!